README.md

```diff
@@ -42,6 +42,11 @@ git clone git@github.com:MasherJames/Ride-My-Way.git
 - Flaskrestfull
 - Virtualenv
 
+## _Enviroment variables_
```

Comment:
Give a dummy example to explain how what a Secret key should be and how the Postgres database U.R.L should look like.